aws/scenarios/vm/os/os.go

```diff
@@ -2,42 +2,24 @@ package os
 
 import (
 	"github.com/DataDog/test-infra-definitions/aws"
-)
-
-type Architecture string
-
-const (
-	AMD64Arch = Architecture("x86_64")
-	ARM64Arch = Architecture("arm64")
-)
-
-type OSType int
-
-const (
-	WindowsOS OSType = iota
-	UbuntuOS         = iota
-	MacOS            = iota
+	"github.com/DataDog/test-infra-definitions/common/os"
 )
 
 type OS interface {
+	os.OS
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Is this a field part of an interface ? Didn't know we could have fields in interfaces

Reply:
`os.OS` is an embedded interface. To implement `OS`, a type must implement the method from `OS` and the methods from `os.OS`.